test/utils/runCodeServerCommand.ts

```diff
@@ -0,0 +1,13 @@
+import { exec } from "child_process"
+import { promisify } from "util"
+
+/**
+ *
+ * A helper function for integration tests to run code-server commands.
+ */
+export async function runCodeServerCommand(argv: string[]): Promise<{ stdout: string; stderr: string }> {
+  const CODE_SERVER_COMMAND = process.env.CODE_SERVER_PATH || "/var/tmp/coder/code-server/bin/code-server"
```

Comment:
Should we default to `path.resolve("../../release-standalone/bin/code-server")`?  Reason being the current path will only work in v1 of Coder.

Reply:
Oh yeah, I was first writing this in v1 hence the default value. Good call.